### PR TITLE
Sample-wise Stochastic Depth Method Card

### PR DESCRIPTION
**Pull request by Landanjs**
Update sample-wise stochastic depth card to new format.

After looking at other cards, it seems like a lot more could be added in some sections, but wait for the future 🙂

### CODE REVIEW

File: composer/algorithms/blurpool/README.md [hunk 1]
```diff
@@ -31,7 +31,8 @@ def training_loop(model, train_loader):
                       optimizers=opt,
                       replace_convs=True,
                       replace_maxpools=True,
-                      blur_first=True)
+                      blur_first=True,
+                      optimizers=opt)
```
Comment (hanlint) [inline composer/algorithms/blurpool/README.md:35]: duplicate argument here

File: composer/algorithms/stochastic_depth/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# 🎰 Stochastic Depth (Sample-wise)
```
Comment (hanlint) [inline composer/algorithms/stochastic_depth/README.md:1]:
```suggestion
# 🎰 Stochastic Depth (Sample)
```

shorten this so no multi-line in the menu -- should do the same for Block

**Review by hanlint — APPROVED** [review]

File: composer/algorithms/blurpool/README.md (hunk 1 above)
Comment (Landanjs) [reply composer/algorithms/blurpool/README.md]: Ooops...

File: composer/algorithms/stochastic_depth/README.md (hunk 2 above)
Comment (Landanjs) [reply composer/algorithms/stochastic_depth/README.md]: Done

**Pull request merged**